### DISCUSSION TITLE
calling restic inside a command hook

### DISCUSSION THREAD

**Discussion by WERTBON1789** [garethgeorge/backrest#366] [Q&A]
I would like to get the timestamp of the latest backup from the restic repository (regardless if created by backrest, or manually created). The easiest way to get this, would be to export the RESTIC_* environment variables of the repository the plan is using for the script to use, and let the user deal with getting the values from restic, they want.
Would anything speak agaist this?

**Comment by garethgeorge** [comment]
Very interesting use case, this feels like something that command hooks should be able to support in my mind.

The big question is, as you call out, the security implications of forwarding the env by default. This might need to be an option that's toggleable.

**Reply by WERTBON1789** [reply]
I think a opt-in toggle with a notice about possible security implications would be great.
